doc/source/commands/files.rst

```diff
@@ -264,20 +281,17 @@ RUN only works when the filename is a bareword filename. It cannot use expressio
                      // called "ProgName.ksm" or "ProgName.ks", when it sees this,
                      // rather than "MyProgram".
 
-    The reasons for the exception to how filenames work for the RUN
-    command are too complex to go into in large detail here. Here's the
-    short version: While the kOS system does defer the majority of the
-    work
-    of actually compiling subprogram scripts until run-time, it still
-    has to
-    generate some header info about them at compile time, and the
-    filename
-    has to be set in stone at that time. Changing this would require a
-    large re-write of some of the architecture of the virtual machine.
+The reasons for the exception to how filenames work for the RUN command are
+too complex to go into in large detail here. Here's the short version: While
+the kOS system does defer the majority of the work of actually compiling
+subprogram scripts until run-time, it still has to generate some header info
+about them at compile time, and the filename has to be set in stone at that
+time. Changing this would require a large re-write of some of the architecture
+of the virtual machine.
```

Comment:
prev formatting was putting this text inside the codeblock monospace font formatting.